src/main/java/io/openio/sds/proxy/ProxyClient.java

```diff
@@ -1066,7 +1067,8 @@ public void deleteContainerProperties(OioUrl url, String... keys) {
     public void deleteContainerProperties(RequestContext reqCtx, OioUrl url,
             String... keys) {
         checkArgument(null != url, INVALID_URL_MSG);
-        checkArgument(null != keys && 0 < keys.length);
+        if (keys == null)
+            keys = new String[0];
```

Comment:
You should create a static `EMPTY_ARRAY` ?
(Creating an empty array still requires a delete in Java ?)